src/main/include/shooter/turret/TurretSubsystem.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc2/command/SubsystemBase.h>
+
+#include <units/velocity.h>
+#include <units/angular_velocity.h>
+
+#include <rmb/motorcontrol/sparkmax/SparkMaxVelocityController.h>
+
+#include <Constants.h>
+
+class TurretSubsystem : public frc2::SubsystemBase {
+ public:
+ /**
+  * Creates a TurretSubsystem with the given wheelDiameter
+  * @param wheelDiameter the diameter of the wheel in meters
+  */
+  TurretSubsystem(units::length::meter_t wheelDiameter = 0_m);
```

Comment:
Linear position has no real meaning for the turret since it spins in place. You can probably just leave everything in radians.

Reply:
fixed in [6e877f5](https://github.com/rambunction4330/RobotCode2022/pull/98/commits/6e877f5a450c059dd696d9591aa208ace3c644ae)